### DISCUSSION TITLE
Allow input_select helper items to be inserted or reordered in the UI

### DISCUSSION THREAD

**Discussion by j9brown** [home-assistant/frontend#12126] [Accessibility] [answered]
I use many input_select helpers to control aspects of my home automation.  Often I want to insert an item at the beginning or middle of the list of drop-down items or rename an existing item.

Unfortunately, the configuration UI only supports adding new items to the end of the list.  So when I want to make a change further up the list, I have to delete the element and all of its successors then type them back in one at a time.

One solution to this problem might be to add an affordance to allow items in the input_select helper dropdown configuration UI to be dragged and reordered.

**Comment by matthiasdebaat** [comment, marked as answer]
Same as https://github.com/home-assistant/frontend/discussions/10106. There has been some work in progress on this, but isn't fixed yet https://github.com/home-assistant/frontend/issues/10107#issue-1011937944
